src/main/java/dev/emi/trinkets/mixin/ClientPlayerEntityMixin.java

```diff
@@ -32,6 +34,6 @@ public void closeScreen(CallbackInfo info){
 	@Redirect(at = @At(value = "INVOKE", target = "Lnet/minecraft/client/network/ClientPlayerEntity;getEquippedStack(Lnet/minecraft/entity/EquipmentSlot;)Lnet/minecraft/item/ItemStack;"), method = "tickMovement")
 	public ItemStack getEquippedStackProxy(ClientPlayerEntity player, EquipmentSlot slot){
 		TrinketComponent comp = TrinketsApi.getTrinketComponent(player);
-		return comp.getStack("chest:cape");
+		return comp.getStack(SlotGroups.CHEST + ':' + Slots.CAPE);
```

Comment:
This method call is still concatenating strings rather than using the function that was added like the others.

Reply:
Resolved in 3092963